.github/workflows/tests.yml

```diff
@@ -23,7 +23,13 @@ jobs:
         run: |
           cache-util restore cargo_git cargo_registry sccache yarn_cache
           cache-util restore aurora-engine-target@${{ matrix.net }}net@${{ hashFiles('**/Cargo.lock') }}:target
-      - run: make test-${{ matrix.net }}net
+      - run: make ${{ matrix.net }}net-test-build
+      - name: List directories
+        run: |
+          ls -la target/wasm32-unknown-unknown/release
+          ls -la
+      - run: cargo test --features ${{ matrix.net }}net-test
+#      - run: make test-${{ matrix.net }}net
 #      - run: rm -rf target/solidity_build/
```

Comment:
LGTM!

NB: let's not commit comments.

Reply:
Sorry, it's already merged now.
I leave this comment because this changes are temporary.